pomodoro/src/main/java/pomodoro/dto/TeamDetailDto.java

```diff
@@ -0,0 +1,65 @@
+package pomodoro.dto;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import pomodoro.entity.Pomodoro;
+import pomodoro.entity.Team;
+import pomodoro.entity.User;
+
+public class TeamDetailDto {
+
+    private Long teamId;
+    private String name;
+    private List<UserDto> users = new ArrayList<UserDto>();
+
+    public TeamDetailDto(Long teamId, String name) {
+        this.teamId = teamId;
+        this.name = name;
+    }
+
+    public TeamDetailDto() {
+    }
+
+    public TeamDetailDto(Team team) {
+        this.teamId = team.getTeamId();
+        this.name = team.getName();
+        for(User user : team.getUsers()) {
+            UserDto userDto = new UserDto();
+            userDto.setEmail(user.getEmail());
+            userDto.setFirstName(user.getFirstName());
+            userDto.setLastName(user.getLastName());
+            for(Pomodoro pomodoro : user.getPomodoro()){
```

Comment:
Please create a constructor in PomodoroDto which accepts the Pomodoro class and contains the lines below.